docs(api-reference): promote `url` instead of `content` (#3499)
scalar/scalar: packages/fastify-api-reference/README.md

})
```

> With [@fastify/swagger], we’re picking it up automatically, so this would be enough:

Reviewer: Wording suggestion (optional)

```suggestion
With [@fastify/swagger] we automatically detect the specification, so all you need is:
```
